### PR TITLE
add quant_aware and quant_post tutorial

### PR DESCRIPTION
**Pull request by slf12**
add quant_aware and quant_post tutorial

### CODE REVIEW

File: docs/docs/tutorials/quant_aware_tutorial.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        acc1, acc5, loss = exe.run(prog, feed=train_feeder.feed(data), fetch_list=outputs)
+        if iter % 100 == 0:
+            print('test', acc1.mean(), acc5.mean(), loss.mean())
+        res[0].append(acc1.mean())
+        res[1].append(acc5.mean())
+        iter += 1
+    print('final test result', np.array(res[0]).mean(), np.array(res[1]).mean())
+```
+
+训练分类网络
```
Comment (wanghaoshuang) [inline docs/docs/tutorials/quant_aware_tutorial.md]: 请说明下训练了多少epoch, 为什么训练这么多epoch。

File: docs/docs/tutorials/quant_aware_tutorial.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+
+```python
+train(train_program)
+```
+
+    ('train', 0.0703125, 0.4765625, 2.7081475)
```
Comment (wanghaoshuang) [inline docs/docs/tutorials/quant_aware_tutorial.md]: 输出可以删掉，或者说明下这是执行上述代码后的输出？

File: docs/docs/tutorials/quant_aware_tutorial.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+    iter = 0
+    res = [[], []]
+    for data in train_reader():
+        acc1, acc5, loss = exe.run(prog, feed=train_feeder.feed(data), fetch_list=outputs)
+        if iter % 100 == 0:
+            print('test', acc1.mean(), acc5.mean(), loss.mean())
+        res[0].append(acc1.mean())
+        res[1].append(acc5.mean())
+        iter += 1
+    print('final test result', np.array(res[0]).mean(), np.array(res[1]).mean())
```
Comment (wanghaoshuang) [inline docs/docs/tutorials/quant_aware_tutorial.md]: log里可以写下"top1_acc="和"top5_acc="

File: docs/docs/tutorials/quant_aware_tutorial.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+    ('test', 0.96875, 1.0, 0.06669818)
+    ('test', 0.96875, 1.0, 0.067689285)
+    ('test', 0.9765625, 1.0, 0.061369315)
+    ('test', 0.953125, 1.0, 0.14121476)
+    ('final test result', 0.971855, 0.9992822)
+
+
+## 4. 量化
+
+按照默认配置在``train_program``和``val_program``中加入量化和反量化op.
```
Comment (wanghaoshuang) [inline docs/docs/tutorials/quant_aware_tutorial.md]: 默认配置是什么，给个API链接？

File: docs/docs/tutorials/quant_aware_tutorial.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+```python
+test(val_quant_program)
+```
+
+    ('test', 0.984375, 1.0, 0.038413208)
+    ('test', 0.9765625, 1.0, 0.045030694)
+    ('test', 0.984375, 1.0, 0.043231864)
+    ('test', 0.984375, 1.0, 0.05136764)
+    ('test', 0.9765625, 1.0, 0.122583434)
+    ('final test result', 0.98505944, 0.999783)
```
Comment (wanghaoshuang) [inline docs/docs/tutorials/quant_aware_tutorial.md]: 还会写一个保存模型的教程么？如果没的话，请在这里补充介绍下如何保存模型。

File: docs/docs/tutorials/quant_post_tutorial.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+```python
+import paddle.dataset.mnist as reader
+train_reader = paddle.batch(
+        reader.train(), batch_size=128, drop_last=True)
+test_reader = paddle.batch(
+        reader.train(), batch_size=128, drop_last=True)
+train_feeder = fluid.DataFeeder(inputs, fluid.CPUPlace())
+```
+
+### 3.2 训练和测试
```
Comment (wanghaoshuang) [inline docs/docs/tutorials/quant_post_tutorial.md]: 建议概述下当前章节是干什么的，以及为什么要做这些工作。

File: docs/docs/tutorials/quant_post_tutorial.md [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+
+    [u'save_infer_model/scale_0',
+     u'save_infer_model/scale_1',
+     u'save_infer_model/scale_2']
+
+
+
+## 4. 离线量化
+
+使用10个batch的数据进行参数校正
```
Comment (wanghaoshuang) [inline docs/docs/tutorials/quant_post_tutorial.md]: 注意使用标点符号。

File: docs/docs/tutorials/quant_post_tutorial.md [hunk 8]
```diff
@@ -0,0 +1,166 @@
...
+    2020-02-05 03:33:53,840-INFO: update the program ...
+    2020-02-05 03:33:53,840-INFO: update the program ...
+
+
+加载量化后的模型进行测试
+
+
+```python
+quant_post_prog, feed_target_names, fetch_targets = fluid.io.load_inference_model(
+        dirname='./quant_post_model',
```
Comment (wanghaoshuang) [inline docs/docs/tutorials/quant_post_tutorial.md]: 要提醒用户这里的`quant_post_model`来自哪里。
虽然有些信息在API文档里说的很详细了，但是在教程文档里，还是要提醒（或简要解释）一下用户可能忽视的重点内容。

**Review by wanghaoshuang — COMMENTED** [review]
建议补充些引导用户的话术，适当解释下某些用户可能会疑惑的地方。

File: docs/docs/tutorials/quant_aware_tutorial.md (hunk 2 above)
Comment (slf12) [reply docs/docs/tutorials/quant_aware_tutorial.md]: done

File: docs/docs/tutorials/quant_aware_tutorial.md (hunk 3 above)
Comment (slf12) [reply docs/docs/tutorials/quant_aware_tutorial.md]: done

File: docs/docs/tutorials/quant_aware_tutorial.md (hunk 4 above)
Comment (slf12) [reply docs/docs/tutorials/quant_aware_tutorial.md]: done

File: docs/docs/tutorials/quant_aware_tutorial.md (hunk 5 above)
Comment (slf12) [reply docs/docs/tutorials/quant_aware_tutorial.md]: done

File: docs/docs/tutorials/quant_post_tutorial.md (hunk 6 above)
Comment (slf12) [reply docs/docs/tutorials/quant_post_tutorial.md]: done

File: docs/docs/tutorials/quant_post_tutorial.md (hunk 7 above)
Comment (slf12) [reply docs/docs/tutorials/quant_post_tutorial.md]: done

File: docs/docs/tutorials/quant_post_tutorial.md (hunk 8 above)
Comment (slf12) [reply docs/docs/tutorials/quant_post_tutorial.md]: done

**Review by wanghaoshuang — APPROVED** [review]
LGTM

**Comment by itminner** [comment]
多卡训练最好补充下，buildstrategy有两个需要关闭的策略
